pkg/cloudprovider/vsphere/cloud.go

```diff
@@ -131,7 +131,7 @@ func (vs *VSphere) Initialize(clientBuilder cloudprovider.ControllerClientBuilde
 	}
 	err = vs.nsxtConnectorMgr.AddSecretListener(vs.informMgr.GetSecretInformer())
 	if err != nil {
-		klog.Warning("Adding NSXT secret listener failed: %v", err)
+		klog.Warningf("Adding NSXT secret listener failed: %v", err)
```

Comment:
Could you please help explain this change? I noticed several `Info` -> `Infof`

Reply:
Arguments are handled in the manner of `fmt.Printf`. Because we bumped klog in this PR, compiling complains a error to `klog.Warning`

Reply:
Got it! Thanks yifeng!